books/scss/mixin.scss

```diff
@@ -0,0 +1,5 @@
+@mixin shadow($property) {
```

Comment:
This feels more like a SASS template than a mixin 🤔

Reply:
I agree, his card does not need this mixin, but I will leave it for a future card.